script/test-versions

```diff
@@ -5,10 +5,6 @@
 set -e
 
 >&2 echo "Running lint checks"
-docker run --rm \
-  ${GIT_VOLUME} \
-  --entrypoint="tox" \
-  "$TAG" -e pre-commit
```

Comment:
Yes, definitely by mistake. I have Tox running locally, but for some reason when running `script/test` tox failed to even start. I'll undo that and squash.